src/server/routes/readings.js

```diff
@@ -32,53 +32,6 @@ function formatBarReadings(rows) {
 	return rows.map(row => [row.start_timestamp.valueOf(), row.reading_sum]);
```

Comment:
I think the function `formatBarReadings` and `formatLineReadings` are no longer necessary. Do you think we should remove them? I can't make comments above this line so I put it here.

Reply:
I agree. The new use is in unitReadings.js where formatLineReadings is now formatReadingRow but formatBarReading is the same name.